nx.json

```diff
@@ -349,6 +343,27 @@
           "filterByTags": ["publishable"]
         }
       }
+    },
+    {
+      "plugin": "@nx/eslint",
+      "options": {
+        "config": "{projectRoot}/eslint.config.js"
+      }
+    },
+    {
+      "plugin": "@nx/vite/plugin",
+      "options": {
```

Comment:
I don't believe these plugins are needed. Introducing inferred tasks shouldn't be in scope of this PR, I believe.

Reply:
removed!